src/components/BannerFallback/HalvingBanner.tsx

```diff
@@ -0,0 +1,53 @@
+import { useMemo } from 'react'
+import classnames from 'classnames'
+import styles from './index.module.scss'
+import SimpleButton from '../SimpleButton'
+import { useAppState } from '../../contexts/providers'
+import { useCountdown } from '../../utils/hook'
+
+export const HalvingBanner = () => {
+  const { statistics } = useAppState()
+
+  const HALVING_EPOCH = 8760
+  const currentEpoch = Number(statistics.epochInfo.epochNumber)
+  const NextHalvingCount = Math.ceil(currentEpoch / HALVING_EPOCH) + 1
+  const targetEpoch = HALVING_EPOCH * NextHalvingCount
+  const singleEpochAverageTime = Number(statistics.estimatedEpochTime)
+  const currentEpochUsedTime =
+    (Number(statistics.epochInfo.index) / Number(statistics.epochInfo.epochLength)) * singleEpochAverageTime
+
+  const estimatedTime = (targetEpoch - currentEpoch) * singleEpochAverageTime - currentEpochUsedTime
+
+  const estimatedDate = useMemo(() => new Date(new Date().getTime() + estimatedTime), [])
+  const [days, hours, minutes, seconds] = useCountdown(estimatedDate)
+
+  const count = () => {
```

Comment:
Not sure about the render result here because I can't open preview page for now, but seems the page needs to display all days/hours/minites/seconds info, think about a scenario, when count down time is:  2 hours 0 minite 2 seconds, the render result is like `2 Hours 2 Seconds`?missing minite info here, and 2 seconds later, the minite panel pops out again.

Reply:
The preview page is at https://twitter.com/lai_jia_wei/status/1702291249800331485?s=52&t=7VGheHqYGBhnWC7OlwdNlQ

Reply:
The preview page is at https://ckb-explorer-frontend-in-magickbase-repo-git-6c6b8e-magickbase.vercel.app/halving

Reply:
Countdown time `2 hours 0 minutes 2 seconds` will be rendered as `2 hours`, not `2 hours 2 seconds`. `minutes` and `seconds` are short-circuited by the `return` in `if (hours > 0)`